src/lib.rs

```diff
@@ -401,13 +401,16 @@
 //! - The `serialize` feature, enabled by default, provides convenience functions for serializing and deserializing with
 //! [serde](https://serde.rs/).
 //!
-//! - The `u32_backend` and `u64_backend` features are re-exported from
+//! - The `rand` feature, enabled by default, re-exports [rand] for your convenience.
+//!
+//! - The backend features are re-exported from
 //! [curve25519-dalek](https://doc.dalek.rs/curve25519_dalek/index.html#backends-and-features) and allow for selecting
-//! the corresponding backend for the curve arithmetic used. The `u64_backend` feature is included as the default.
+//! the corresponding backend for the curve arithmetic used. The `ristretto_u64` feature is included as the default.
+//! Other features are mapped as `ristretto_u32`, `ristretto_fiat_u64` and `ristretto_fiat_u32`.
 //!
-//! - The `simd_backend` feature is re-exported from
+//! - The `ristretto_simd` feature is re-exported from
 //! [curve25519-dalek](https://doc.dalek.rs/curve25519_dalek/index.html#backends-and-features) and enables parallel formulas,
-//! using either AVX2 or AVX512-IFMA. This will automatically enable the `u64_backend` and requires Rust nightly.
+//! using either AVX2 or AVX512-IFMA. This will automatically enable the `ristretto_u64` and requires Rust nightly.
```

Comment:
I'm not sure if the fact that we renamed the backend features can be documented better, English is my second language. Alternatively we could rename them back.